### ISSUE TITLE
tf.nn.sigmoid and tf.nn.softmax diff?

### ISSUE THREAD

**Issue by ronyuzhang** [ailias/Focal-Loss-implement-on-Tensorflow#4]

**Comment by ChiefGodMan** [comment]
The paper of He proposed sigmoid has more robust training effect than softmax, But he didn't explain why.  You can have a try.
I think maybe it depends the dependent of each probability of result, softmax may have more small probability values(all sum is 1), which may cause more negative samples.
